tests/helperDb.js

```diff
@@ -14,7 +13,7 @@ var courseDB = require('../lib/course-db');
 var postgresqlUser = 'postgres';
 var postgresqlDatabase = 'pltest';
 var postgresqlHost = 'localhost';
-var initConString = 'postgres://localhost/postgres';
+var initConString = 'postgres://postgres@localhost/postgres';
```

Comment:
I seem to remember this being necessary to work on my machine, I'll take another look though.